api/client/run.go

```diff
@@ -74,6 +74,7 @@ func (cli *DockerCli) CmdRun(args ...string) error {
 		flDetach     = cmd.Bool([]string{"d", "-detach"}, false, "Run container in background and print container ID")
 		flSigProxy   = cmd.Bool([]string{"-sig-proxy"}, true, "Proxy received signals to the process")
 		flName       = cmd.String([]string{"-name"}, "", "Assign a name to the container")
+		flDetachKeys = cmd.String([]string{"-detach-keys"}, "", "Set up escape key sequence")
```

Comment:
Still not sure if "Set up escape key sequence" is the best wording, but no really good alternative ("Set escape key sequence for detach",  "Set detach key sequence"?

@moxiegirl perhaps you have some suggestions?